ega-charts/localega/templates/verify-deploy.yaml

```diff
@@ -29,6 +29,7 @@ spec:
         image: "{{ .Values.verify.repository }}:{{ .Values.verify.imageTag }}"
         imagePullPolicy: {{ .Values.verify.imagePullPolicy | quote }}
         command: ["gosu", "lega", "ega-verify"]
+        args: ["--log", "debug"]
```

Comment:
if we are going to use this args we should have for most services. We already use it for `verify` and `mapper`, maybe `ingest` as well. and I think the `debug` value should come from: https://github.com/NBISweden/LocalEGA-helm/blob/master/ega-charts/localega/values.yaml#L7 where possible values are one of `'default', 'debug', 'logstash', 'silent'`

Reply:
Yes, we should only need to specify this in one place

Reply:
The java services takes the `ENV`, python services via `args:` exceptions is inbox since that one uses an entrypoint script.